run_unittests.py

```diff
@@ -37,8 +37,9 @@
 from mesonbuild.mesonlib import is_linux, is_windows, is_osx, is_cygwin, windows_proof_rmtree
 from mesonbuild.mesonlib import python_command, meson_command, version_compare
 from mesonbuild.environment import Environment
+from mesonbuild.environment import Environment, detect_ninja
```

Comment:
__[Flake8]__

[F811] redefinition of unused 'Environment' from line 39


<!-- Comment by [SideCI](https://sideci.com) -->